docs/create-and-manage-employee.md

```diff
@@ -0,0 +1,138 @@
+# Manage Employees
+
+## Create New Employee
+
+1. The user opens the Contacts Module ->clicks ‘Add’ and selects ‘Employee’;
+1. The system will display the ‘New Employee’ screen:
+     1. ‘First name’ filed- required;
+     1. ‘Middle name’ screen;
+     1. ‘Last name’ field- required;
+     1. ‘Full name’ field- required;
+     1. Widgets:  
+         1. Addresses;
+         1. Emails;
+         1. Phones.
+     1. ‘Employee type’ field;
+     1. ‘Birthday- date picker;
+     1. ‘Organization(s)’ drop down;
+     1. ‘Time zone’ drop down;
+     1. ‘Default language’ field;
+     1. ‘Photo URL’ field;
+     1. ‘Create’ button disabled by default until the required fields are filled out.
+1. The user should fill out the form and click the ‘Create’ button;
+1. The system will create the new employee and display it on the ‘Contacts’ list.
+
+**Important**: If the Organization was selected, the new Employee will appear on the Organization’s contact list. If no Organization was selected, the new employee will appear on the ‘Organizations & Contacts’ list of the Contacts Module.
+
+![Fig. Add New Employee](media/screen-add-new-employee.png)
+
+![Fig. Add Employee](media/screen-add-employee.png)
+
+## Edit Employee
+
+### Add Account to Employee
+
+1. On ‘Employee details’ screen the user selects the ‘Accounts’ widget;
+1. The system will first display the ‘Manage customer accounts’ screen;
+     1. If any accounts were added before, they will be displayed on the screen;
+     1. The ‘Add’ button is always active;
+     1. The ‘Delete’ button is disabled by default and becomes active only when at least one account from the list is selected.  
+1. The user clicks the ‘Add’ button;
+1. The system will open the ‘New Account’ screen and prompt the user to select a store for the new account. The list of available stores will be displayed on the screen;
+1. The user selects the store;
+1. The system will open the ‘Manage customer account’ screen and prompt the user to fill out the form:
+     1. ‘Login’ field- required;
+     1. ‘Email’ field;
+     1. ‘Password’ field- required;
+     1. ‘Repeat password’ field- required;
+     1. ‘Assign roles’ - optional. The user can select the roles to be assigned to the employee using the check boxes; ![Fig.Assign Roles](media/screen-assign-roles.png)
+     1. ‘Create’ button is disabled by default and becomes active when ALL required fields are filled out;
+1. The user fills out the form and clicks the ‘Create’ button;
+1. The system will create the new account and display it on ‘Manage customer accounts’ screen. The User information will be displayed on the right side on a separate blade.
+
+### Edit User Information
+
+On User Information screen all fields and widgets are editable.
+
+#### Edit Account Type
+
+1. The user navigates to User Information screen and selects the account type from the drop down list;
+1. The system provides three default types :
+
+     1. Customer;
+     1. Manager;
```

Comment:
Here we can tell that only Manager and Administrator account types have access to admin tool. Customer do not have access here.

Reply:
You do not need to remove Customer here. Need to note that Customer do not have access to admin tool. It could be done as in other places:
Important:
lalala